.github/workflows/continuous-integration.yml

```diff
@@ -93,11 +93,12 @@ jobs:
       - name: Annotate ESLint results
         run: yarn run eslint --ext .js --ext .jsx  --format ./script/github-actions/eslint-annotation-format.js .
 
-      - name: Run Sass lint
-        run: yarn run sass-lint --format json --verbose --output sasslint-report.json
+      - name: Run Stylelint
+        run: yarn run stylelint verbose --output-file stylelint-report.json --formatter json src/**/*.scss
 
-      - name: Annotate Sass lint results
-        run: node ./script/github-actions/sasslint-annotation-format.js
+      - name: Annotate Stylelint results
+        if: ${{ always() }}
```

Comment:
Always flag enabled as stylelint is more strict and errors out if a lint if violated